arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -524,50 +605,42 @@
     "SubscriptionIdParameter": {
       "name": "subscriptionId",
       "in": "path",
-      "description": "The subscription ID that identifies an Azure subscription.",
+      "description": "The Azure subscription ID.",
       "required": true,
-      "type": "string"
+      "type": "string",
+      "x-ms-parameter-location": "client"
     },
     "ApiVersionParameter": {
       "name": "api-version",
       "in": "query",
-      "description": "The API version to use for the request.",
+      "description": "Client Api Version.",
       "required": true,
-      "x-ms-parameter-location": "client",
-      "type": "string"
+      "type": "string",
+      "x-ms-parameter-location": "client"
     },
     "ResourceGroupParameter": {
       "name": "resourceGroupName",
       "in": "path",
-      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+      "description": "The name of the resource group that contains the resource.",
```

Comment:
I believe the description here does not match with other specs under the same composite swagger, either revert this to the older description or update all the descriptions under the composite swagger to match this. It would break builds otherwise. 
The problem here is that when we merge the documents into a single model, its common nodes should be exactly the same, it would be difficult to say which one overrides all other nodes.
A similar issue was filed [here](https://github.com/Azure/azure-rest-api-specs/issues/1071#issuecomment-290552215)